README.md

```diff
@@ -1,23 +1,19 @@
-# Next.js + Jest
+# Sparkbox Apprentice Page 
 
-This example shows how to configure Jest to work with Next.js.
+## Technical Overview 
 
-This includes Next.js' built-in support for Global CSS, CSS Modules, and TypeScript!
+#### Tech Stack:
 
-## How to Use
+- [Next.js](https://nextjs.org/)
+- [Sass Modules](https://nextjs.org/docs/basic-features/built-in-css-support#sass-support)
+- [Jest](https://jestjs.io/)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)
 
-Quickly get started using [Create Next App](https://github.com/vercel/next.js/tree/canary/packages/create-next-app#readme)!
+#### CI/CD:
 
-In your terminal, run the following command:
+- [Netlify](https://docs.netlify.com/?_gl=1%2a9cbevu%2a_gcl_aw%2aR0NMLjE2MzcxNzk2MTAuQ2p3S0NBaUE3ZEtNQmhCQ0Vpd0FPX2NyRkpkcTd0Uk1VQk9feHF4dHIwRENEZTZvSjdDRXo2dGNCUm5sVGo2cUE5NHJWVXluR05CdGt4b0N6ZzBRQXZEX0J3RQ..&_ga=2.212470036.1605752707.1637179610-1559964017.1632236220&_gac=1.85689835.1637179610.CjwKCAiA7dKMBhBCEiwAO_crFJdq7tRMUBO_xqxtr0DCDe6oJ7CEz6tcBRnlTj6qA94rVUynGNBtkxoCzg0QAvD_BwE) is used for automatic deployments and deployment previews. 
```

Comment:
Let's link to the dashboard for this project for the Netlify link: https://app.netlify.com/sites/sb-apprentices/overview

Reply:
Could we also update the links to use [footnote syntax](https://github.com/sparkbox/standard/tree/main/code-style/markdown#style) instead please?

Reply:
Just updated the READMe to use footnote syntax and changed the link to the dashboard for the project!